docs/real-world-examples/ccfraud.md

```diff
@@ -0,0 +1,35 @@
+# Finance example for Federated Learning in Azure ML
+
+This is a short example where we showcase possibilities of using Azure Machine Learning(AML) for training a model for credit card fraud detection in federating learning fashion while using variety of models. This is common scenario for credit card companies, banks and othe similar financial institutions.
```

Comment:
First sentence is very long and could be split in 2. Typo in second sentence ("othe").